packages/taxdat/R/get_data_fidelity.R

```diff
@@ -21,7 +21,15 @@ get_data_fidelity <- function(cache,config,cholera_directory){
   dimnames(actual_cases) <- dimnames(modeled_cases_chain_mean)
   modeled_cases_chain_mean <- reshape2::melt(modeled_cases_chain_mean)
   actual_cases <- reshape2::melt(actual_cases)
-  actual_cases$censoring <- rep(stan_data$censoring_inds, each = nchain)
+  if(nrow(actual_cases) != length(rep(stan_data$censoring_inds, each = nchain))){
```

Comment:
@KaiyueZou Do you know why the data fidelity code had to suddenly change? Under what circumstances is this statement tru?
@javierps Is this related to dropping censored obs that are equal to 0?

Reply:
I made this change because suddenly the MWI country data report has an error that complains the number of rows of the `actual_cases` don't match with the number of `stan_data$censoring_inds` while other countries don't have this issue.

Reply:
This is maybe because of the 0 censored data dropping indeed. 

We should be dropping things from sf_cases_resized,  so I don't see why the sizes of the objects should be different. This is suspicious.

@KaiyueZou could you look into an example of the data objects that are used here to see what observations/modeled cases are missing?

Reply:
@javierps Hello Javier, I checked `stan_data$y`, which gives the number of observed cases for each observation after calculating the mean across chains and has the data for both tfrac adjusted cases and censored cases, each taking up 50% of the rows. The number of rows of chain-averaged `stan_data$y` matched with the length of `stan_data$censoring_inds` after its NAs being removed. Do you want me to check other data object, like sf_cases_resized? Could you clarify? Thanks! And here's the modified code: https://github.com/HopkinsIDD/cholera-mapping-pipeline/commit/96b1a3788d96da6b68cb95fcf40ecdc9abd8de03